easybuild/easyblocks/x/xmipp.py

```
@@ -0,0 +1,161 @@
+        java_root = get_software_root('Java')
+        if not java_root:
+            self.log.error("Java not loaded as a dependency, which is required for %s" % self.name)
+
+        # extract some dependencies that we really need and can't find anywhere else
+        # alglib tarball has version in name, so lets find it with a glob
+        # we can't do this in extract step before these are in the original sources tarball, so we need to know
+        # startdir first
+        external_path = os.path.join(self.cfg['start_dir'], 'external')
+        alglib_tar = glob.glob(os.path.join(external_path, 'alglib*.tgz'))[0]
+        for src in ['bilib.tgz', 'bilib.tgz', 'condor.tgz', alglib_tar, 'scons.tgz']:
+            extract_file(os.path.join(external_path, src), external_path)
+
+        # make sure we start in the start dir
+        os.chdir(self.cfg['start_dir'])
+
+        # build step expects these to exist
+        mkdir(os.path.join(self.cfg['start_dir'], 'bin'))
+        mkdir(os.path.join(self.cfg['start_dir'], 'lib'))
+
+        cmd = ' '.join([
+            self.cfg['preconfigopts'],
+            'python external/scons/scons.py',
+            'mode=configure',
+            '-j {parallel}',
+            '--config=force',
+            'profile=no',
+            'fast=yes',
+            'warn=no',
+            'release=yes',
+            'gtest=no',
+            'cuda=no',
+            'debug=no',
+            'matlab=no',
+            'java=no',
+            'LINKERFORPROGRAMS="$CXX"',
+            'MPI_BINDIR="{mpi_bindir}"',
+            'JAVA_HOME="{java_home}"',
+            'JAVAC=javac',
+            'CC="$CC"',
+            'CXXFLAGS="%s"' % ' '.join([
+                '$CXXFLAGS',
+                '-DMPICH_IGNORE_CXX_SEEK',
+                '-I$EBROOTPYTHON/include/python{short_python_ver}',
+                '-I$EBROOTPYTHON/{python_libdir}/numpy/core/include/',
```

```python
        '-I%s' % os.path.join(python_root, python_libdir, 'numpy', 'core', 'include'),
```

And actually, you should make sure this path is there, and fail hard if it's not.